src/config.js

```diff
@@ -328,6 +331,7 @@ export default class Config {
     this.enableMetaFolder = Boolean(this.getOption('enable-meta-folder'));
     this.enableLockfileVersions = Boolean(this.getOption('yarn-enable-lockfile-versions'));
     this.linkFileDependencies = Boolean(this.getOption('yarn-link-file-dependencies'));
+    this.packBuiltPackages = Boolean(this.getOption('pack-built-packages'));
```

Comment:
Shouldn't the option be called `experimental-build-mirror` or something like this, to clearly state that it's experimental?

Reply:
Yeah, makes sense.
How about `experimental-pack-script-packages-in-mirror`?